ConsistencyManager/ConsistencyManager.swift

```diff
@@ -416,12 +416,13 @@ public class ConsistencyManager {
         if garbageCollectionInterval > 0 {
             // We're going to use a low priority queue for this timer
             // We need to use a dispatch queue because NSOperationQueue doesn't support delays
-            let dispatchQueue = dispatch_get_global_queue(DISPATCH_QUEUE_PRIORITY_LOW, 0)
+            
+            let dispatchQueue = DispatchQueue.global(qos: DispatchQoS.QoSClass.default)
             // Weak here is necessary, otherwise, we'd have a retain cycle.
-            dispatch_after(dispatch_time(DISPATCH_TIME_NOW, Int64(UInt64(garbageCollectionInterval) * NSEC_PER_SEC)), dispatchQueue) { [weak self] in
+            dispatchQueue.asyncAfter(deadline: DispatchTime.now() + Double(Int64(UInt64(garbageCollectionInterval) * NSEC_PER_SEC)) / Double(NSEC_PER_SEC)) { [weak self] in
```

Comment:
You added an extra ```/ Double(NSEC_PER_SEC)``` from the previous diff. Was this intentional? Created #28 btw, we can look at this later (don't need to block this).